Dockerfile

```diff
@@ -2,6 +2,7 @@ ARG base
 FROM golang:1.12.4-alpine as builder
 RUN apk add make git curl
 ADD . /nkn
+ADD ./certs /nkn
```

Comment:
We should add it in the 2nd stage, see https://docs.docker.com/develop/develop-images/multistage-build/ for more information

Reply:
done